test/cli/commands_test.py

```diff
@@ -924,6 +924,17 @@ def test_cli_disable_noqa_flag():
     assert r"L:   5 | P:  11 | L010 |" in raw_output
 
 
+def test_cli_get_config():
```

Comment:
```suggestion
def test_cli_get_default_config():
    """`nocolor` and `verbose` values loaded from config if not specified via CLI."""
```
Linting is failing since we need a docstring here.

(for future reference would recommend using the pre-commit hooks to help catch these at commit time: `tox -e pre-commit -- install` 😄)